doc/api/next_api_changes/removals/26885-AD.rst

```diff
@@ -0,0 +1,4 @@
+``raw``
+~~~~~~~
+
+... of `.get_grid_positions` is removed without replacements.
```

Comment:
This works because probably nothing else has this method name, but I'd prefer it be a bit more specific just for anyone reading the original reST file.
```suggestion
``raw`` parameter
~~~~~~~~~~~~~~~~~

... of `.GridSpecBase.get_grid_positions` is removed without replacements.
```

Reply:
Done, thank you!